apiary.apib

```diff
@@ -2720,22 +2715,22 @@ Retrieves a list of study templates that exist in a particular organization.
 
         [
             {
-                "ID": "NURALOGIX_GENERIC",
+                "ID": "76f59b35-0e5f-4d19-8563-6891g5091d1e",
                 "StatusID": "ACTIVE",
-                "Name": "Generic Study",
-                "Description": "Study that can be used for all purposes",
-                "BundleID": null,
+                "Name": "ANURA_AGW",
+                "Description": "Anura Template with AGW points",
                 "Config": {},
                 "StudyTypeID": "GENERAL",
+                "Created": 1648564820,
                 "Signals": [
-                    "BP_SYSTOLIC",
-                    "BP_DIASTOLIC",
-                    "HEART_RATE",
-                    "MENTAL_STRESS_INDEX",
-                    "SNR",
-                    "STRESS_SCORE"
+                "BP_SYSTOLIC",
+                "BP_DIASTOLIC",
+                "HEART_RATE",
+                "MENTAL_STRESS_INDEX",
+                "SNR",
+                "STRESS_SCORE"
```

Comment:
You can leave the "Signals" format as it was before.

Reply:
HEART_RATE, "MENTAL_STRESS.." should be deprecated